emcc.py

```diff
@@ -918,6 +922,24 @@ def detect_fixed_language_mode(args):
       if separate_asm:
         shared.Settings.SEPARATE_ASM = os.path.basename(asm_target)
 
+      if 'EMCC_STRICT' in os.environ:
```

Comment:
this happens after the code above for `-DEMSCRIPTEN`, so it seems it would only work for the setting and not the environment var?